plugins/inputs/dnac/README.md

```diff
@@ -0,0 +1,78 @@
+# Cisco DNAC Input Plugin
+
+The Cisco DNAC plugin gathers metrics about the health of the network monitored
+by DNA Assurance.
+
+## Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage <plugin-name>`.
+
+```toml
+[[inputs.dnac]]
+  ## Specify DNAC Base URL
+  dnacbaseurl = "sandboxdnac.cisco.com"
+  ## Specify Credentials
+  username = "devnetuser"
+  password = "Cisco123!"
+  ## Debug true/false
+  debug = "false"
+  ## SSL Verify true/false
+  sslverify = "false"
+  ## Report Client Health
+  clienthealth = true
+  ## Report Network Health
+  networkhealth = true

```

Comment:
We like to avoid booleans wherever possible as it leads to long lists and more difficult logic to maintain. I would prefer you have something like the following (feel free to bikeshed it):

```
## Health to report
report = ["network", "client"]
```

This way if you ever wish to expand the collection, it does not require yet another boolean.

Reply:
ok